### ISSUE TITLE
Support fo:inline-container

### ISSUE THREAD

**Issue by lionelmoi** [dita-ot/dita-ot#2507]
The DITA-OT embedded FOP version does not support the fo:inline-container.

OxygenXML seems to using an amended version that can render the element.

**Comment by raducoravu** [comment]
Yes, Oxygen uses Apache FOP 2.1 for producing the PDF using DITA OT and the default downloaded DITA OT distribution still comes with Apache FOP 1.1 (but it can be manually uploaded to Apache FOP 2.1).

**Comment by dfemia** [comment]
Are there plans to move the DITA-OT to FOP 2.1?

**Comment by infotexture** [comment]
See #2399.

**Comment by dfemia** [comment]
Thanks!
